utils/entity.js

```diff
@@ -137,8 +137,7 @@ function prepareEntityForTemplates(entityWithConfig, generator) {
             : entityWithConfig.entityStateName
     );
 
-    entityWithConfig.reactiveRepositories =
-        entityWithConfig.reactive && ['mongodb', 'cassandra', 'couchbase', 'neo4j'].includes(entityWithConfig.databaseType);
+    entityWithConfig.reactiveRepositories = entityWithConfig.reactive;
```

Comment:
I think (but probably in another PR since this once is quite big) that we should kill this `reactiveRepositories` config parameter, since it's now equivalent to  `reactive`

Reply:
Yes, definitely!